backend/src/main/kotlin/com/tngtech/apicenter/backend/config/SecurityConfiguration.kt

```diff
@@ -25,12 +26,33 @@ class SecurityConfiguration : WebSecurityConfigurerAdapter() {
             .csrf().disable()
             .authorizeRequests()
             .antMatchers("/sessions").permitAll()
-            .and()
-            .authorizeRequests()
+
+            .antMatchers("/").permitAll()
+            .antMatchers("/3rdpartylicenses.txt").permitAll()
+            .antMatchers("/favicon.ico").permitAll()
+            .antMatchers("/index.html").permitAll()
+            .antMatchers("/main.js").permitAll()
+                .antMatchers("/main.js.map").permitAll()
+            .antMatchers("/open-iconic.*").permitAll()
+            .antMatchers("/polyfills.js").permitAll()
+                .antMatchers("/polyfills.js.map").permitAll()
```

Comment:
Do we really need to open the `map` files for everyone? I think they are just for developers, right?

Reply:
This is partially true. I was guided by what console log messages indicated a resource script could not be loaded with a 403 response. Some of them are certainly needed, I listed all for safe measure, but can be a bit more thorough